app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,32 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = current_user.foods
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = Food.new(food_params.merge(user_id: current_user.id))
+
+    if @food.save
+      redirect_to foods_path, notice: 'Food was successfully created.'
+    else
+      render :new, alert: 'Cannot create food, retry again.'
+    end
+  end
+
+  def destroy
+    @food = Food.find(params[:id])
+    @food.destroy
+
+    redirect_to foods_path, notice: 'Food was successfully destroyed.'
```

Comment:
- [ ] @team its good practice to have both success and failure paths for methods that have  create, delete, or update  actions, kindly ensure your methods have both paths.

Reply:
Fixed.